utils/rpms/daos.spec

```diff
@@ -130,19 +130,21 @@ scons %{?no_smp_mflags}              \
       PREFIX=%{?buildroot}%{_prefix}
 BUILDROOT="%{?buildroot}"
 PREFIX="%{?_prefix}"
-sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/TESTING/.build_vars.*
-mv %{?buildroot}%{_prefix}/lib{,64}
+sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{daoshome}/TESTING/.build_vars.*
+sed -i -e s/${BUILDROOT//\//\\/}[^\"]\*/${PREFIX//\//\\/}/g %{?buildroot}%{_prefix}/lib/.build_vars.*
+#mv %{?buildroot}%{_prefix}/lib{,64}
 #mv %{?buildroot}/{usr/,}etc
-mkdir -p %{?buildroot}/%{_exec_prefix}/lib/%{name}
-mv %{?buildroot}%{_prefix}/lib64/daos %{?buildroot}/%{_exec_prefix}/lib/
-mv %{?buildroot}%{_prefix}/{TESTING,lib/%{name}/}
-cp -al ftest.sh src/tests/ftest %{?buildroot}%{daoshome}/TESTING
-find %{?buildroot}%{daoshome}/TESTING/ftest -name \*.py[co] -print0 | xargs -r0 rm -f
+mkdir -p %{?buildroot}/%{_exec_prefix}/lib64/%{name}
+#cp -al VERSION %{?buildroot}/%{_exec_prefix}/lib/%{name}
+mkdir -p %{?buildroot}/%{_exec_prefix}/lib64/%{name}/certgen
+#mv %{?buildroot}%{_prefix}/{TESTING,lib/%{name}/}
+#cp -al ftest.sh src/tests/ftest %{?buildroot}%{daoshome}/TESTING
+#find %{?buildroot}%{daoshome}/TESTING/ftest -name \*.py[co] -print0 | xargs -r0 rm -f
 #ln %{?buildroot}%{daoshome}/{TESTING/.build_vars,.build_vars-Linux}.sh
-mkdir -p %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/daos_api.py %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/daos_cref.py %{?buildroot}%{daoshome}/utils/py
-cp -al src/utils/py/conversion.py %{?buildroot}%{daoshome}/utils/py
+#mkdir -p %{?buildroot}%{daoshome}/utils/py
+#cp -al src/utils/py/daos_api.py %{?buildroot}%{daoshome}/utils/py
```

Comment:
Is this a temporary change?  If not, just remove the lines instead of commenting.   We can tell what changed in the diff.

Reply:
yes, will remove those commented lines since this PR passed all CI tests now.